helm/cluster-api-cleaner-cloud-director/values.yaml

```diff
@@ -17,15 +17,20 @@ pod:
 
 # Add seccomp to pod security context
 podSecurityContext:
+  runAsNonRoot: true
   seccompProfile:
     type: RuntimeDefault
-  runAsNonRoot: true
 
 # Add seccomp to container security context
-securityContext:
```

Comment:
This naming is added by @giantswarm/team-shield and it is the same for all repositories. Are we sure about changing it?

Reply:
This currently isn't managed by automation or something like that (would be really cool if it was though). Renaming it is fine